be/src/exec/pipeline/exchange/exchange_sink_operator.h

```diff
@@ -67,20 +68,26 @@ class ExchangeSinkOperator final : public Operator {
 private:
     class Channel;
 
-    const std::shared_ptr<SinkBuffer>& _buffer;
+    static const int32_t default_driver_sequence = 0;
```

Comment:
The static constant variable maybe better uses upper case, like `DEFAULT_DRIVER_SEQUENCE`.

Reply:
OK